app/services/activity_creator.rb

```diff
@@ -31,13 +31,11 @@ def create_completed_audit_activities_observation
   end
 
   def create_activity_observation
-    points = @activity.activity_type.score_when_recorded_at(@activity.school, @activity.happened_on)
```

Comment:
Was this intentional? Might have missed some other change, but with the FeatureFlag off doesn't this mean schools aren't scoring points?

Reply:
It's now in a call back on observation. Tidied it to be behave in the same way as activities